packages/aria-voyager/src/controls/menu.ts

```diff
@@ -58,7 +58,17 @@ export class Menu extends Control {
   }
 
   readItems() {
-    this.items = [...this.element.querySelectorAll('& > [role="menuitem"]')] as HTMLElement[];
+    // Find all descendant elements with role "menuitem", "menuitemcheckbox", or "menuitemradio"
+    const items = this.element.querySelectorAll(
+      '[role="menuitem"], [role="menuitemcheckbox"], [role="menuitemradio"]'
```

Comment:
oh dang, I really forgot tha latter two here :( Thanks for adding the selectors here, too.